src/main/java/hwr/oop/ToDoList.java

```diff
@@ -0,0 +1,136 @@
+package hwr.oop;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Comparator;
+import java.util.Objects;
+
+public class ToDoList {
+    private String name;
+    private List<ToDoItem> listToDos;
+    private String fileName;
+    private List<Bucket> buckets;
+
+    public ToDoList(String name) {
+        this(name, null);
+    }
+
+    public ToDoList(String name, String fileName) {
+        this.name = name;
+        this.listToDos = new ArrayList<>();
+        this.fileName = fileName;
+        this.buckets = new ArrayList<>();
+    }
+
+    //setter
+    public void setFileName(String fileName) {
+        this.fileName = fileName;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setListToDos(List<ToDoItem> listToDos) {
+        this.listToDos = listToDos;
+    } //needs test?
```

Comment:
Bitte als TODO markieren.
```suggestion
    } //TODO: Needs test?
```